### PR TITLE
Add Logstash & syslog-ng integration documentation

### PR DESCRIPTION
**Pull request by otetard**

### CODE REVIEW

File: docs/integrations/transports.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+SEKOIA.IO supports multiple log collectors:
+
+- Syslog over TLS (`intake.sekoia.io:10514`): you can forward your events by using the Syslog protocol specified in RFC 5424.
+- RELP over TLS (`intake.sekoia.io:11514`): your can forward your events by using Rsyslog’s reliable protocol called RELP.
+- HTTPS (`https://intake.sekoia.io`): you can `POST` your JSON events.
+
+This makes SEKOIA.IO compatible with a large number of product and solutions to collect your logs.
+
+Even if Rsyslog is the most supported solution to push your logs to SEKOIA.IO, you can use other solutions, such as Logstash, syslog-ng, Greylog, etc.
```
Comment (celinequivy) [inline docs/integrations/transports.md:11]:
```suggestion
Even if Rsyslog is the most supported solution to push your logs to SEKOIA.IO, you can use other solutions, such as Logstash, syslog-ng, Graylog, etc.
```

**Review by celinequivy — COMMENTED** [review]
mkdocs.yml should be modified too.

File: docs/integrations/transports.md (hunk 1 above)
Comment (otetard) [reply docs/integrations/transports.md]: Fixed, thanks!

**Review by Darkheir — APPROVED** [review]
Awesome documentation :-)

**Review by celinequivy — APPROVED** [review]

**Review by alexandreLavenant — APPROVED** [review]

**Pull request merged**